scripts/tests/prepare_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+if [[ ! -f ".env" ]]
+then
+    echo ".env file does not exist"
+    touch .env
+    cat << 'EOF' > .env
+        export PATH=$PATH:~/Qt/6.2.4/macos/bin:$PATH
```

Comment:
this works on your computer only :) Not on mine where:
- I use a different version of QT
- it's not installed in that path
- and it's not mac...

Reply:
good catch, will remove